### PR TITLE
docs: add blog cncf projects with podman desktop

### PR DESCRIPTION
**Pull request by cdrage**
docs: add blog projects with podman desktop

### What does this PR do?

Adds how we can use Podman Desktop with projects such as Minikube,
Backstage and Dapr.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop-internal/issues/352

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
Ready for review! @Firewall @slemeur 

Although..

**DO NOT MERGE UNTIL 1.15 RELEASE**

This relies on the "port forwarding" feature of 1.15 as I've included it as a screenshot so that we can port forward / view Backstage components without having to create an Ingress.

File: website/blog/2024-11-15-cncf-projects.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Podman Desktop serves as a powerful tool for managing and visualizing cloud-native applications and can interact seamlessly with a range of [CNCF (Cloud Native Computing Foundation)](https://www.cncf.io/) projects. 
+
+It's an accessible platform for developers working with single-container applications, multi-container configurations with Compose files, and complex, distributed applications on Kubernetes clusters. 
+
+## Key features of Podman Desktop for CNCF projects
+
+Podman Desktop brings together three powerful features for managing small to large-scale projects:
+* **Container Management**: Supports creating, running, and monitoring containers.
+* **Compose Support**: Allows you to deploy applications defined in [Compose files](https://www.compose-spec.io/). This is particularly useful for managing applications that require multiple services, such as web servers, databases, and caches.
+* **Kubernetes Integration**: Offers tools to manage multi-node Kubernetes clusters, making it ideal for handling more complex distributed applications that need orchestration across several pods and services.
```
Comment (slemeur) [inline website/blog/2024-11-15-cncf-projects.md]: I'd add that we can also setup local kubernetes environment with the Kind and Minikube extensions. WDYT?

**Review by slemeur — APPROVED** [review]

File: website/blog/2024-11-15-cncf-projects.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+The following video provides a complete guide from installation to cluster creation:
+
+<ReactPlayer playing playsinline controls url='https://github.com/containers/podman-desktop-media/raw/refs/heads/minikube/video/guide.mp4' width='100%' height='100%' />
+
+## Backstage
+
+[Backstage](https://backstage.io/) is an open-source platform for building developer portals, designed by Spotify. It empowers engineering teams to create customized, centralized hubs for managing and documenting their services, applications, and infrastructure. Backstage’s extensible architecture includes features for cataloging software components, organizing documentation, managing cloud resources, and tracking workflows.
+
+A popular method for deploying Backstage is through a [Helm chart](https://github.com/backstage/charts), Once deployed, you can view Backstage’s services in the Kubernetes Dashboard to monitor components and ensure proper configuration:
```
Comment (Firewall) [inline website/blog/2024-11-15-cncf-projects.md]:
```suggestion
A popular method for deploying Backstage is through a [Helm chart](https://github.com/backstage/charts). Once deployed, you can view Backstage’s services in the Kubernetes Dashboard to monitor components and ensure proper configuration:
```

**Review by Firewall — APPROVED** [review]

**Comment by cdrage** [comment]
Updated and thanks for the reviews!

Big note: **DO NOT MERGE UNTIL AFTER 1.15 release!** since there are port-forwarding screenshots in this post.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by benoitf** [comment]
draft as it needs to wait

**Comment by cdrage** [comment]
@benoitf 1.15 is going out shortly, good to merge? I've updated the file name to reflect today's date.

**Comment by benoitf** [comment]
yes

**Pull request merged**